terraform/vars.tf

```diff
@@ -13,6 +13,13 @@ variable "region" {
   type = string
 }
 
+#=================================================
+#  EMAILS NOTIFICATIONS
+#=================================================
+variable "new_user_notification_email_address" {
+  type = string
+}
+
```

Comment:
No seeing a similar snippet in `cool/vars.tf` as well.

Reply:
Resolved